app/routes/index.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+    store: Ember.inject.service(),
+    currentUser: Ember.inject.service(),
+
+    model() {
+        return this.get('store').query('preprint-provider', {
+            'filter[permissions]': 'view_review_logs'
```

Comment:
Is there a way to do `OR`s in the API?

Reply:
Not that I can see... both `filter[permissions]=view_review_logs,set_up_moderation` and `filter[permissions]=view_review_logs&filter[permissions]=set_up_moderation` behave like an `AND`. Since admin permissions are a superset of moderator permissions, though, we could just query for `view_review_logs` and filter by `set_up_moderation` in the browser...

Reply:
Oh wait, I wrote the comma part, it's not in every filter... I made it `AND` because it seemed more consistent with the rest of the API. But looking again, the only other place a comma-separated list happens is for a list of guids, where it's an `OR`. I guess I should make `filter[permissions]=view_review_logs,set_up_moderation` be an `OR`